raw-spans-grouper/helm/templates/raw-spans-grouper-config.yaml

```diff
@@ -20,7 +20,13 @@ data:
       {{- if .Values.rawSpansGrouperConfig.kafka.streams.config.sessionTimeoutMs }}
       session.timeout.ms = {{ int .Values.rawSpansGrouperConfig.kafka.streams.config.sessionTimeoutMs }}
       {{- end }}
-
+      commit.interval.ms = {{ int .Values.rawSpansGrouperConfig.kafka.streams.config.commitIntervalMs }}
```

Comment:
Shouldn't this be also under if condition, otherwise build might fail when this value not present right ?

Reply:
Yes, unless default value does not work great for hypertrace I'd rather use conditional.